src/redux/slices/QuranReader/translations.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+const DEFAULT_TRANSLATIONS = ['20'];
```

Comment:
We've two default translations of English. 

20: Shaih International 
131:  Clear Quran

Reply:
I like that you used the ids instead of the names for the translations. It's more robust.

That should be a `number` not a `string` to match the api, right? You'll have to refactor it in a couple of places

Reply:
> We've two default translations of English.
> 
> 20: Shaih International
> 131: Clear Quran

Okay got it, I will include 131 as well. The reason I included 20 is because the default value of the translations set inside`DEFAULT_VERSES_PARAMS` was set to 20 only.

Reply:
> I like that you used the ids instead of the names for the translations. It's more robust.
> 
> That should be a `number` not a `string` to match the api, right? You'll have to refactor it in a couple of places

Okay got it. The reason I have it as a `string` is because when the selected option(s) changes, the type of the value inside each `option.value` inside `e.target.selectedOptions` is `string`. But no problem I can convert it to Number before setting it in the store.